docs/test_framework_integration/spock.md

```diff
@@ -1,9 +1,10 @@
-# TestContainers-Spock
-[Spock](https://github.com/spockframework/spock) extension for [TestContainers](https://github.com/testcontainers/testcontainers-java) library, which allows to use Docker containers inside of Spock tests.
+# Spock
 
-# Usage
+[Spock](https://github.com/spockframework/spock) extension for [Testcontainers](https://github.com/testcontainers/testcontainers-java) library, which allows to use Docker containers inside of Spock tests.
 
-## @Testcontainers class-annotation
+## Usage
```

Comment:
The annotation lives in a separate module (unlike JUnit 4 integration), worth adding module coordinates here

Reply:
👍